.pre-commit-config.yaml

```diff
@@ -0,0 +1,41 @@
+ci:
+  autofix_commit_msg: "ci(pre-commit): autofix"
+  autoupdate_commit_msg: "ci(pre-commit): autoupdate"
```

Comment:
You can activate pre-commit CI here, which can automatically fix errors.
https://pre-commit.ci/

Reply:
Thank you! I'd like to add the following.
```
    autoupdate_schedule: weekly
```

Reply:
It's okay, but I think it's not necessary since it's a default value.

Reply:
Thank you, and removed default value.  d3b8812d41e27120669e77d0a298e543378c9d11